### PR TITLE
add Yoda

### PR DESCRIPTION
**Pull request by whoisandie**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ Some good apps written with Electron.
 - [Cumulus](https://github.com/gillesdemey/Cumulus) - SoundCloud player in your menubar.
 - [We Build SG](https://github.com/webuildsg/osx) - Upcoming events & recently updated repos from webuild.sg.
 - [Piglet](https://github.com/jenslind/piglet) - Run Grunt tasks from the menubar.
+- [Yoda](https://github.com/whoisandie/yoda) - Browse and download youtube videos
```
Comment (sindresorhus) [inline readme.md]: > Browse and download YouTube videos.

**Comment by shinnn** [comment]
-1

[Terms of Service](https://www.youtube.com/static?template=terms&gl=US)

> You shall not download any Content unless you see a “download” or similar link displayed by YouTube on the Service for that Content. You shall not copy, reproduce, distribute, transmit, broadcast, display, sell, license, or otherwise exploit any Content for any other purposes without the prior written consent of YouTube or the respective licensors of the Content.

**Comment by sindresorhus** [comment]
@shinnn I know, but that's not our problem. I still think the app is useful regardless.

**Comment by whoisandie** [comment]
Thats what I think too cuz there are a lot of downloaders out there & this is just another wrapper around node-ytdl-core.

File: readme.md (hunk 1 above)
Comment (sindresorhus) [inline readme.md]: @whoisandie :arrow_up:

**Comment by arthurvr** [comment]
Thanks. I squashed your commits.

**Pull request closed without merging**